tests/unit/src/test/scala/org/scalafmt/tests/PatSuite.scala

```diff
@@ -22,4 +22,5 @@ object PatSuite extends BaseScalaPrinterTest {
   checkEnumerator("a = b")
   checkCase("case `a` :: `b` :: _ =>")
   checkCase(""" case a b `c` =>""")
+  checkCase("case _ op (a | b) =>")
```

Comment:
Any test case for `with` change?

Reply:
Good catch. We need to add test coverage.

Reply:
https://github.com/olafurpg/scala-syntax/pull/15#discussion_r171026847